runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperConfig.java

```diff
@@ -129,18 +139,127 @@ default boolean menuOption()
 		keyName = "subscriptionFilter",
 		name = "Show subscription types",
 		description = "Only show free worlds, member worlds, or both types of worlds in sidebar",
-		position = 8
+		section = typeFilters,
+		position = 7
 	)
 	default SubscriptionFilterMode subscriptionFilter()
 	{
 		return SubscriptionFilterMode.BOTH;
 	}
 
+	@ConfigItem(
+		keyName = "PVPTypeFilter",
+		name = "Show PVP",
+		description = "Consider PVP worlds",
+		section = typeFilters,
+		position = 8
+	)
+	default boolean PVPTypeFilter()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "bountyTypeFilter",
+		name = "Show Bounty",
+		description = "Consider Bounty worlds",
+		section = typeFilters,
+		position = 9
+	)
+	default boolean bountyTypeFilter()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "skillTotalTypeFilter",
+		name = "Show Skill Total",
+		description = "Consider Skill Total worlds",
+		section = typeFilters,
+		position = 10
+	)
+	default boolean skillTotalTypeFilter()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "highRiskTypeFilter",
+		name = "Show High Risk",
+		description = "Consider High Risk worlds",
+		section = typeFilters,
+		position = 11
+	)
+	default boolean highRiskTypeFilter()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "lastManStandingTypeFilter",
+		name = "Show Last Man Standing",
+		description = "Consider Last Man Standing worlds",
+		section = typeFilters,
+		position = 12
+	)
+	default boolean lastManStandingTypeFilter()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "tournamentTypeFilter",
+		name = "Show Tournament",
+		description = "Consider Tournament worlds",
+		section = typeFilters,
+		position = 13
+	)
+	default boolean tournamentTypeFilter()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "deadmanTypeFilter",
+		name = "Show Deadman",
+		description = "Consider Deadmam worlds",
+		section = typeFilters,
+		position = 14
+	)
+	default boolean deadmanTypeFilter()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "leagueTypeFilter",
+		name = "Show League",
+		description = "Consider League worlds",
+		section = typeFilters,
+		position = 15
+	)
+	default boolean leagueTypeFilter()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "exclusiveTypeFilter",
+		name = "Exclusive filtering",
+		description = "Only show selected world types",
```

Comment:
This is not clear enough to be understandable. Without looking at the code implemented using this toggle, I cannot understand what behavior this changes.

Reply:
Requested changes done. Hopefully this description is now understandable.